modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -63,10 +63,6 @@ The features mentioned in the following table are currently in Technology Previe
 |1.10.0
 |NA
 
-|Argo Rollouts
-|1.9.0
-|NA
-
 |ApplicationSet Progressive Rollout Strategy
 |1.8.0
 |NA
```

Comment:
For this entry in this table: 
```
|Argo CD applications in non-control plane namespaces
|1.7.0
|NA
```

Is this the "Argo CD Applications in any namespace" feature? Shouldn't it be removed from this table? And I think we should refer to it in the RN text in the same way, if that is the feature :+1:  (If it is not the feature, is it missing from this TP table?)

Reply:
No, this is not related to that. It is something else. Let me check this further.

Reply:
This is the same feature referred as 'Applications in any namespace' in upstream ArgoCD(https://argocd-operator.readthedocs.io/en/latest/usage/apps-in-any-namespace/). There was a discussion and I think it was decided to continue with `Argo CD applications in non-control plane namespaces` @Srivaralakshmi can confirm.

Reply:
This feature is getting GA-ed, that might be the reason why it is missing in the TP table.